src/main/java/org/trigon/hardware/misc/leds/AddressableLEDStrip.java

```diff
@@ -65,46 +65,47 @@ void clearLEDColors() {
     }
 
     @Override
-    void blink(Color firstColor, Color secondColor, double blinkingIntervalSeconds) {
+    void blink(Color firstColor, double speed) {
```

Comment:
Speed in what unit? Maybe this will help me understand the `correctedSpeed` better?

Reply:
The CANdle animations use a speed between 0 and 1. I don't know what units. But I know that the higher the number the higher the speed. So I adapted the AddressableLED commands to also use speed in those units.